flit/validate.py

```diff
@@ -0,0 +1,335 @@
+"""Validate various pieces of packaging data"""
+
+import logging
+import os
+from pathlib import Path
+import re
+import requests
+import sys
+
+from .common import InvalidVersion
+
+log = logging.getLogger(__name__)
+
+def get_cache_dir() -> Path:
+    """Locate a platform-appropriate cache directory for flit to use
+
+    Does not ensure that the cache directory exists.
+    """
+    if os.name == 'posix' and sys.platform != 'darwin':
+        # Linux, Unix, AIX, etc.
+        # use ~/.cache if empty OR not set
+        xdg = os.environ.get("XDG_CACHE_HOME", None) or (
+        os.path.expanduser('~/.cache'))
```

Comment:
Hard to read indentation (IMHO)

Reply:
Is the new commit better? I'm not quite sure what was making it tricky.

Reply:
It's just no indentation after the `or (\n`, looks to my eyes (brain?), like a missing assignment. At least I don't easily see the it's part of the above assignment. It may be britany's brain flakes are of low quality.

Reply:
Ah, I see it now. It was so misleading I didn't even notice what it was meant to be ;-)